internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -212,7 +212,8 @@ EnvironmentManagerRole:
           Action: [
             "ec2:DescribeSubnets",
             "ec2:DescribeSecurityGroups",
-            "ec2:DescribeNetworkInterfaces"
+            "ec2:DescribeNetworkInterfaces",
+            "ec2:DescribeRouteTables"
```

Comment:
I don't see why it is required in this PR 🤔

Reply:
Because of subnetLister who lists the VPC subnets calls DescribeRouteTables

Reply:
Oh ok gotcha